system-test/Dockerfile.nodenode-alpine

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.17-alpine as builder
```

Comment:
Title of this file seems off (Dockerfile.nodenode-alpine)

Would "Dockerfile.node-alpine" work?